README.md

```diff
@@ -226,6 +227,36 @@ if (!userInfo) {
 }
 ```
 
+## Routing strategies
```

Comment:
Let's take this one step back. Why are you routing? Because you have chosen to get tokens by redirect. So I think the top level topic is "strategies for obtaining tokens". There we mention, popup, signin widget, and redirect. Then under redirect, we start getting into the routing, for handling the callback.